src/tools/file-base.r

```diff
@@ -218,6 +220,7 @@ extensions: [
     + BMP ../extensions/bmp/ext-bmp.c [BMP] _
     + Locale ../extensions/locale/ext-locale.c [Locale] ../extensions/locale/ext-locale-init.reb
     + UUID ../extensions/uuid/ext-uuid.c [UUID] ../extensions/uuid/ext-uuid-init.reb
+    + ODBC ../extensions/odbc/ext-odbc.c [ODBC] ../extensions/odbc/ext-odbc-init.reb
```

Comment:
Extensions are also in [make-make.r](https://github.com/metaeducation/ren-c/blob/master/src/tools/make-make.r#L543). By default, they will be builtin, unless the user chooses not to by specifying in the config file or the command line.